sig-ibmcloud/charter.md

```diff
@@ -0,0 +1,33 @@
+# SIG IBMCloud Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+The IBMCloud SIG enables activities and discussion around building, deploying, maintaining, supporting,
+and using Kubernetes, and related CNCF projects, on IBM Public and Private Clouds.
+
+The authoritative source for SIG information is the [SIG README] file.
+
+### Out of scope
+
+* Discussion around commercial aspects of IBM Public and Private clouds.
+
+## Roles and Organization Management
+
+This SIG adheres to the Roles and Organization Management outlined in [sig-governance]
+and opts-in to updates and modifications to [sig-governance], with exception that the SIG only
+has the Chair role at present.
```

Comment:
Does this mean there are no duties for a Tech Lead to perform? Or that the Chair satisfies the duties of Tech Lead?

Reply:
Yes, the chair role satisfies the duties as of today. We don't have code repo etc yet but once we have we will rethink on adding Tech lead role. Hope makes sense.

Reply:
Can add a sentence stating this explicitly? eg: "Chairs must also fulfill all of the responsibilities of the Tech Lead role as outlined in [sig-governance]" (with a link)